hadoop-hdds/interface-admin/src/main/proto/ScmAdminProtocol.proto

```diff
@@ -322,6 +328,28 @@ message DatanodeUsageInfoResponseProto {
   repeated DatanodeUsageInfoProto info = 1;
 }
 
+/*
+  Datanode disk balancer report request message.
+*/
+message DatanodeDiskBalancerReportRequestProto {
+  optional uint32 count = 1;
```

Comment:
Please use one request and one response to wrap the two getDiskBalancerInfo API.